### PR TITLE
plugging through database import issues

### PR DESCRIPTION
**Pull request by rshorey**
NOT ACTUALLY READY TO GO, want to run a bunch of stuff past @fgregg 

1) this was failing locally for me on the changes you made to the legistar scraper. I'll comment directly on that line. Can you check it and make sure my changes don't hurt anything?

2) I'm having trouble with duplicate events during the database import phase. In some cases it looked like event rescheduling notices, so I added some more possible rescheduling text, but it's also pulling things like this, which all seem to be the same event.
https://chicago.legistar.com/MeetingDetail.aspx?ID=184426&GUID=0FB72168-5E0C-4F42-B782-E79DCCC1B9B6&Options=info|
https://chicago.legistar.com/MeetingDetail.aspx?ID=190991&GUID=14BD8DD5-4C57-4A11-AA09-D124C9551DB0&Options=info|
https://chicago.legistar.com/MeetingDetail.aspx?ID=191000&GUID=F46D8B14-13E1-4500-A47D-428604931403&Options=info|

How should we deal with those? Our system thinks they're duplicates, so won't successfully import.

I'm running bills now locally to see if there are any remaining import issues there, if I can get it to finish on my local.

### CODE REVIEW

File: chicago/legistar.py [hunk 1]
```diff
@@ -30,7 +30,8 @@ def pages(self, url, payload=None) :
         yield page
 
         next_page = page.xpath("//a[@class='rgCurrentPage']/following-sibling::a[1]")
-        del payload['ctl00$ContentPlaceHolder1$btnSearch']
+        if payload and 'ctl00$ContentPlaceHolder1$btnSearch' in payload:
+            del payload['ctl00$ContentPlaceHolder1$btnSearch']
```
Comment (rshorey) [inline chicago/legistar.py]: was failing locally for me here on people and events

Comment (fgregg) [inline chicago/legistar.py]: Could you post traceback please.

Comment (fgregg) [inline chicago/legistar.py]: nevermind, got it.

Comment (fgregg) [inline chicago/legistar.py]: This is fine!

**Comment by fgregg** [comment]
Ugh. So the finance committee has their normal meeting: https://chicago.legistar.com/MeetingDetail.aspx?ID=191000&GUID=F46D8B14-13E1-4500-A47D-428604931403&Options=info%7C

During which they will have two public hearings: 
https://chicago.legistar.com/MeetingDetail.aspx?ID=184426&GUID=0FB72168-5E0C-4F42-B782-E79DCCC1B9B6&Options=info%7C
https://chicago.legistar.com/MeetingDetail.aspx?ID=190991&GUID=14BD8DD5-4C57-4A11-AA09-D124C9551DB0&Options=info%7C

I'm not sure the right way to model this. Can we figure out a way to hack around this right now,and track this as a separate issue?

**Comment by rshorey** [comment]
Should I skip anything with "public hearing" under location for now? Will that cause many more terrible problems? And can you file the relevant issue, please?

**Comment by fgregg** [comment]
Try the skipping 'public hearing', and I'll file an issue.

**Comment by rshorey** [comment]
@fgregg with these commits, this code is now running and importing successfully on my local. Can you take a look and make sure it doesn't look like I'm doing anything totally out of step with what you had in mind? If it looks OK, I'll pull it in and kick off a fresh run tomorrow AM.

File: chicago/__init__.py [hunk 2]
```diff
@@ -7,8 +7,8 @@
 
 class Chicago(Jurisdiction):
     division_id = 'ocd-division/country:us/state:il/place:chicago'
-    classification='council'
-    name = 'Chicago City Council'
+    classification='legislature'
+    name = 'Chicago'
```
Comment (fgregg) [inline chicago/__init__.py]: This is the Chicago City Council, what's wrong with calling it that?

**Comment by fgregg** [comment]
Looks good. Why change the name from "Chicago City Council" to "Chicago"?

File: chicago/__init__.py (hunk 2 above)
Comment (rshorey) [inline chicago/__init__.py]: I'm generally a little confused about jurisdiction vs organization naming. I just ran this by @paultag and he said the jurisdiction should be "chicago city government" and that the classification should be government. I'm going to make that fix and set the whole thing up to run locally before I head out tonight so I can be sure that doesn't screw anything up. Sound OK?

Comment (fgregg) [inline chicago/__init__.py]: Sounds great!

**Comment by rshorey** [comment]
OK, this ran successfully last night on my local, so I'm going to merge and try it in production!

**Comment by fgregg** [comment]
Woot!

**Comment by rshorey** [comment]
Yes! @fgregg if more development/checking needs to happen here, it'd probably make sense for you to get the database running locally. I'm doing this on a vagrant box using this repo: https://github.com/opencivicdata/vagrant-opencivicdata.org

**Comment by fgregg** [comment]
yah! seems like the project needs to get to a place where the whole
kit-and-caboodle is not necessary for people to contribute scrapers, but
I'm definitely a special flower and will do that.

**Pull request merged**